rfd/xxxx-config-resource-docs.md

```diff
@@ -0,0 +1,554 @@
+---
+authors: Paul Gottschling (paul.gottschling@goteleport.com)
+state: draft
+---
+
```

Comment:
Could you a required approvers section?

https://github.com/gravitational/teleport/blob/master/rfd/0000-rfds.md?plain=1#L106